skin-deep.js

```diff
@@ -18,6 +19,27 @@ function shallowRender(elementOrFunction, context) {
       var node = findNodeIn(shallowRenderer, query);
       return node && node.props && String(node.props.children);
     },
+    text: function() {
+      function getTextFromNode(node) {
+        // console.log(node);
+        if (typeof node.type === 'function') {
+          return '<' + node.type.displayName + ' />';
+        }
+        if (typeof node === 'string') return node;
+
+        if (typeof node.map === 'function') {
```

Comment:
Hrm, that should only make `node.props.children` an array, which you handle below